### DISCUSSION TITLE
Welcome to Partisan-Telegram Discussions!

### DISCUSSION THREAD

**Discussion by wrwrabbit** [wrwrabbit/Partisan-Telegram-Android#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wrwrabbit** [comment]
@ivanovivan12589 @vivabelarus here we can discuss things outside of issues

**Reply by vivabelarus** [reply]
Да, хорошее место. @ivanovivan12589, как успехи?
